resources/services/apigateway/apigateway_domain_names.go

```diff
@@ -146,6 +160,22 @@ func ApigatewayDomainNames() *schema.Table {
 						Type:        schema.TypeUUID,
 						Resolver:    schema.ParentIdResolver,
 					},
+					{
+						Name:        "arn",
+						Description: "The Amazon Resource Name (ARN) for the resource.",
+						Type:        schema.TypeString,
+						Resolver: client.ResolveARN(client.ApigatewayService, func(resource *schema.Resource) ([]string, error) {
+							r, ok := resource.Item.(types.BasePathMapping)
+							if !ok {
```

Comment:
I think this check is not necessary here as we check it in tests and it's not something user controlled.

Reply:
Removed